tiffany-tiffanyteaze/week-6/my-react-todo/src/components/CreateTodoForm.js

```diff
@@ -0,0 +1,27 @@
+import React, { Component } from 'react'
+
+export default class CreateTodoForm extends Component {
+  state = {
+      todo: ''
+  };
+  
+    render() {
+    return (
+      <div>
+        <form onSubmit={ this.onFormSubmit } id="taskForm">
+            <input
+              onChange={ this.onInputChange }  
```

Comment:
Your issue is here. You have an OnChange, but where is your onInputChange? You need to add that function into this component to update the todo state. Other than that alot of your code is great work. Good job.